x/evm/module.go

```diff
@@ -156,6 +156,7 @@ func (am AppModule) LegacyQuerierHandler(*codec.LegacyAmino) sdk.Querier {
 // module-specific GRPC queries.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterQueryServiceServer(cfg.QueryServer(), am.keeper)
+	cfg.RegisterMigration(types.ModuleName, 1, keeper.GetMigrationHandler(am.keeper, am.nexus))
```

Comment:
why is this registered here?

Reply:
That where cosmos-hub did it so I just assumed that this is the best place for it.